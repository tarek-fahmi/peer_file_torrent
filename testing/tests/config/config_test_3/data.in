Unallowed Config Specs
./testing/bin/btide ./testing/resources/config/neg_peers.cfg
