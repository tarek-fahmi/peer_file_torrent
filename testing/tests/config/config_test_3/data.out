Max peers (-128) outside of permitted range (1 - 2048)